pyproject.toml

```diff
@@ -6,16 +6,16 @@ authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
 fastapi = "^0.65.2"
-uvicorn = {extras = ["standard"], version="^0.11.2"}
+uvicorn = {extras = ["standard"], version=">=0.11.8,<1.0.0"}
```

Comment:
How about using carrots for `uvicorn` and `httpx` for consistency with the format of the other dependencies? (e.g. `uvicorn = {extras = ["standard"], version="^0.11.8"}`)

Reply:
Good idea! With that notation I had to bump up the versions a bit to avoid poetry errors.